www/html/js/webJS.js

```diff
@@ -46,8 +47,102 @@ $(document).ready(function() {
 
 
     });
+
 });
 $(document).ready(function() {
     $(".kt-tabs-component-content").hide();
     $("#aboutsite").show();
-});
\ No newline at end of file
+
+});
+//tracking navbar
+$(document).ready(function() {
+    $("li").click(function() {
+
+        //set global variables
+        var city;
+        var country;
+        var page = $(this).text();
+
+
+        // current date
+        var currentdate = new Date();
+        var datetime = currentdate.getFullYear() + "-" +
+            (currentdate.getMonth() + 1) + "-" +
+            currentdate.getDate() + " " +
+            currentdate.getHours() + ":" +
+            currentdate.getMinutes() + ":" +
+            currentdate.getSeconds();
+
+        //city and country
+        $.get("http://ipinfo.io", function(response) {
+
+            city = response.city;
+            country = response.country;
+
+            //call function
+            updateNavbarTracking(page, datetime, city, country);
+
+        }, "jsonp");
+
+
+
+
+
+
+    });
+});
+//ajax function
+function updateNavbarTracking(page, time, city, country) {
+
+
+
+    $.ajax({
+        url: 'index.php/landing/trackingnavigation',
+        type: 'post',
+        dataType: 'json',
+        data: { 'page': page, 'time': time, 'city': city, 'country': country }
+    });
+}
+
+
+//tracking landings
+$(document).ready(function() {
+    //set global variables
+    var city;
+    var country;
+
+    // current date
+    var currentdate = new Date();
+    var datetime = currentdate.getFullYear() + "-" +
+        (currentdate.getMonth() + 1) + "-" +
+        currentdate.getDate() + " " +
+        currentdate.getHours() + ":" +
+        currentdate.getMinutes() + ":" +
+        currentdate.getSeconds();
+
+    //city and country
+    $.get("http://ipinfo.io", function(response) {
+
+        city = response.city;
+        country = response.country;
```

Comment:
Hmm. Let's refactor this function which gets the city and country by creating a javascript function which calls this get function and returns a javascript object which contains the city and country. In this case you would just return response. 

So it'll look like:
var location = getLocation();
city = location.city;
country = location.country;
updateLandingTracking(datetime, city, country);